platform/darwin/test/MGLStyleValueTests.swift

```diff
@@ -1,8 +1,53 @@
 import XCTest
 import Mapbox
 
-
+#if os(iOS) || os(watchOS) || os(tvOS)
+typealias ColorType = UIColor
```

Comment:
With this typealias in place, it might be possible to unconditionalize the giant `#if` block in `testFunctionsWithDataDrivenProperties()`.